src/BaseProvider.js

```diff
@@ -66,31 +66,8 @@ const ChangeRenderer = (options) => {
 
 
 
-const CurrencyRenderer = ({unit, currency, decimals}) => {
-  unit = unit || 'mBTC';
-
-  const getFormat = (currency) => {
-    /* determined after mtgox api */
-    const front = "%s%v";
-    const back = "%v %s";
-    const frontFormats = {
-      USD: front, CAD: front, AUD: front, GBP: front,
-      HKD: front, NZD: front, SGD: front, THB: front,
-      MXN: front
-    };
-
-    return frontFormats[currency] || back;
-  };
-
-  const changeUnit = (number) => {
-    if (unit === 'mBTC') {
-      return Number(number) / 1000.0;
-    } else {
-      return Number(number);
-    }
-  };
-
-  let format = getFormat(currency);
+const CurrencyRenderer = ({currency, coin, decimals}) => {
```

Comment:
I could look into it, however I'm just not sure how to combine it with altcoins. Maybe you could elaborate what the mBTC feature is supposed to do and how you see it working when BTC is not part of the currency pair. The best I can think of is adding mBTC to the Provider > Coin as an option, which would divide the Indicator value by 1000.